docs/source/model-registry.rst

```diff
@@ -22,7 +22,7 @@ Model
     An MLflow Model is created from an experiment or run that is logged with one of the model flavor’s ``mlflow.<model_flavor>.log_model()`` methods. Once logged, this model can then be registered with the Model Registry.
 
 Registered Model
-    An MLflow Model can be registered with the Model Registry. A registered model has a unique name, contains versions, associated transitional stages, model lineage, and other metadata.
+    An MLflow Model can be registered with the Model Registry. A registered model has a unique name, contains versions, version aliases, version tags, model lineage, and other metadata.
```

Comment:
```suggestion
    An MLflow Model can be registered with the Model Registry. A registered model has a unique name, contains versions, aliases, tags, and other metadata.
```

Minor but not sure it makes sense to mention "version tags" here since we (I think correctly) don't mention the other metadata fields on model versions here, e.g. model version descriptions